lib/hls/hls_parser.js

```diff
@@ -1219,6 +1220,12 @@ shaka.hls.HlsParser = class {
       const [width, height] = resolution ? resolution.split('x') : [null, null];
 
       const videoRange = tag.getAttributeValue('VIDEO-RANGE');
+      let videoLayout = tag.getAttributeValue('REQ-VIDEO-LAYOUT');
+      if (videoLayout == 'CH-STEREO,CH-MONO') {
+        videoLayout = 'CH-STEREO';
+      } else if (videoLayout == 'CH-MONO,CH-STEREO') {
```

Comment:
Incidentally, from that spec, I would also guess the intention is that if no `REQ-VIDEO-LAYOUT` tag is present, we are probably meant to assume the content is `CH-MONO`. Since it specifies that the presence of stereoscopic content requires a `REQ-VIDEO-LAYOUT`, but monoscopic content does not.

Reply:
Yeah, that draft spec you sent me ([link](https://developer.apple.com/streaming/HLS-draft-pantos.pdf) for anyone reading this), backs this up:

> By default a video variant is monoscopic, so an attribute
 consisting entirely of REQ-VIDEO-LAYOUT="CH-MONO" is unnecessary
 and SHOULD NOT be present.

Reply:
Draft spec: https://developer.apple.com/streaming/HLS-draft-pantos.pdf

Reply:
Done!